avro-codegen/build.gradle

```diff
@@ -49,4 +49,89 @@ jar {
 shadowJar {
     //needs to have a classifier to not overwrite the regular jar
     classifier = "all"
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId project.group
+            artifactId project.name
+            version project.version
+
+            from components.java
+            artifact sourceJar
+            artifact javadocJar
+
+            //we strive to meet https://central.sonatype.org/pages/requirements.html
+            pom {
+                name = 'Avro Util'
+                description = 'utilities for writing code that works across major avro versions'
+                url = 'https://github.com/linkedin/avro-util'
+
+                licenses {
+                    license {
+                        name = 'BSD 2-Clause'
+                        url = 'https://raw.githubusercontent.com/linkedin/avro-util/master/LICENSE'
+                    }
+                }
+                developers {
```

Comment:
Have these folks ever contributed to the project 🤣 ...?

Reply:
yes, thayve also contributed to the helper codebase back when it was an internal project and so github history is distorted.
we really need some gradle magic to not repeat this pom blurb so many time though ...